app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", root_path , id: "logo" %>
```

Comment:
Q: `sample app` chỗ này viết thường mà sao trên UI chụp a thấy in hoa vậy em?

Reply:
dạ bởi vì id logo trong file custom.scss có  text-transform: uppercase nên in hoa

Reply:
dạ bởi vì id logo trong file custom.scss có text-transform: uppercase nên in hoa